lib/main_table.json

```diff
@@ -0,0 +1,12 @@
+{
+    "AttributeDefinitions": [
+        {"AttributeName": "index", "AttributeType": "S"}
```

Comment:
key sounds perfect. I wrote this before I read further down and saw the compoundness.